Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -93,6 +96,9 @@ public final class ManagePledgeViewModel:
     self.goToChangePaymentMethod = self.menuOptionSelectedSignal
       .filter { $0 == .changePaymentMethod }
       .ignoreValues()
+
+    self.rewardReceivedViewControllerViewIsHidden = projectAndReward
+      .map { project, reward in reward.isNoReward || project.personalization.backing?.status != .collected }
```

Comment:
Hopefully this is the same logic @eoji was trying to explain to me ... please let me know if I'm off here

Reply:
:chefs_kiss: lgtm!